RELEASE.md

```diff
@@ -1,6 +1,12 @@
 # Release History
 *****************
 
+## Release ONDEWO NLU APIS 1.1.1
+
+### New Features
+ * [OND211-1732] Implement endpoints directly create/update/get/delete training phrases.
```

Comment:
Listing API is not mentioned in the release notes

Reply:
Yes, because I actually didn't update it in this PR - but I add it!